diff --git a/node_modules/@mantine/dropzone/cjs/Dropzone.js b/node_modules/@mantine/dropzone/cjs/Dropzone.js
index a8913f7..50670ff 100644
--- a/node_modules/@mantine/dropzone/cjs/Dropzone.js
+++ b/node_modules/@mantine/dropzone/cjs/Dropzone.js
@@ -135,7 +135,8 @@ const _Dropzone = React.forwardRef((props, ref) => {
     onDragOver,
     onFileDialogCancel,
     onFileDialogOpen,
-    preventDropOnDocument
+    preventDropOnDocument,
+    useFsAccessApi: false
   });
   hooks.assignRef(openRef, open);
   const isIdle = !isDragAccept && !isDragReject;
diff --git a/node_modules/@mantine/dropzone/esm/Dropzone.js b/node_modules/@mantine/dropzone/esm/Dropzone.js
index 4a78a31..a3ddfa8 100644
--- a/node_modules/@mantine/dropzone/esm/Dropzone.js
+++ b/node_modules/@mantine/dropzone/esm/Dropzone.js
@@ -127,7 +127,8 @@ const _Dropzone = forwardRef((props, ref) => {
     onDragOver,
     onFileDialogCancel,
     onFileDialogOpen,
-    preventDropOnDocument
+    preventDropOnDocument,
+    useFsAccessApi: false
   });
   assignRef(openRef, open);
   const isIdle = !isDragAccept && !isDragReject;
